master/test/integration/api/api_trials_test.go

```diff
@@ -124,9 +124,8 @@ func trialDetailAPITests(
 			ctx, _ := context.WithTimeout(creds, 10*time.Second)
 			req := apiv1.GetTrialRequest{TrialId: int32(trial.ID)}
 
-			tlCl, err := cl.GetTrial(ctx, &req)
+			_, err = cl.GetTrial(ctx, &req)
 			assert.NilError(t, err, "failed to fetch api details")
-			assert.Equal(t, tlCl.Trial.WorkloadCount, int32(1), "mismatching workload length")
```

Comment:
missing out on this test

Reply:
the original test prior was looking for `len(tlCl.Trial.Workloads) == 1`, which was replaced by `tlCl.Trial.WorkloadCount == 1`, but now has no equivalent